README.md

```diff
@@ -1,52 +1,66 @@
-<!-- PROJECT LOGO -->
-<br />
-<div align="center">
-  <a href="[https://github.com/othneildrew/Best-README-Template](https://github.com/OSMChina/BusFensi)">
-    <img src="https://github.com/fltb.png" alt="Logo" width="80" height="80">
+ <a href="[https://github.com/othneildrew/Best-README-Template](https://github.com/OSMChina/BusFensi)">
+    <img src="https://github.com/fltb.png" alt="Logo" width="200" height="200">
   </a>
 
-  <h3 align="center">BusFensi</h3>
+# BusFensi
 
-  <p align="center">
-    一个基于前端技术的便于操作的OpenStreetMap中公共交通线路编辑器
-    <br />
-    An Easy-to-Use OpenStreetMap Public Transport Route Editor Based on Front-End Technology
-    <br />
-    <a href=""><strong>Explore the docs » (还没写好，请前往 /docs 目录)</strong></a>
-    <br />
-    <!--
-    <br />
-    <a href="https://github.com/othneildrew/Best-README-Template">View Demo</a>
-    &middot;
-    <a href="https://github.com/othneildrew/Best-README-Template/issues/new?labels=bug&template=bug-report---.md">Report Bug</a>
-    &middot;
-    <a href="https://github.com/othneildrew/Best-README-Template/issues/new?labels=enhancement&template=feature-request---.md">Request Feature</a>
-    -->
-  </p>
-</div>
+##### BusFensi 是一款易于使用的 OpenStreetMap 公共交通路线编辑器，是一款用户友好的工具，旨在简化 OpenStreetMap 上公共交通路线的创建和管理。
 
-<hr/>
+###### <a href="https://github.com/OSMChina/BusFensi/blob/main"><strong>Explore the docs » (还没写好，请前往 /docs 目录)</strong></a>
+------------
```

Comment:
```suggestion
[简体中文](README.zh-Hans.md) | [English](README.en.md) | [日本語](README.ja.md)
------------
```

(JA translation is just as a example, to show how I want to organize docs for languages. I can write a little but not such frequency.)